opm/parser/eclipse/EclipseState/Tables/FlatTable.hpp

```diff
@@ -48,6 +48,43 @@ struct RockTable : public FlatTable< ROCKRecord > {
     using FlatTable< ROCKRecord >::FlatTable;
 };
 
+struct PVCDORecord {
+    static constexpr std::size_t size = 5;
+
+    double reference_pressure;
+    double volume_factor;
+    double compressibility;
+    double viscosity;
+    double viscosibility;
+};
+
+struct PvcdoTable : public FlatTable< PVCDORecord > {
```

Comment:
I suppose here the philosophical question what is considered to be a "table" kicks in again.